src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/PackageFixture.Uac.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+		"id": "dc567365-2812-4bdd-939f-ca9045bb02d2",
```

Comment:
The indentation is inconsistent (tabs vs. spaces).
Also same for previous json file

Reply:
fixed